### PR TITLE
Updated the deliverable section of the charter

### LINKED ISSUE #46: Linked Data Vocabulary as registry
(Originally raised by @samuelweiler in https://github.com/w3c/strategy/issues/262#issuecomment-822696701; moved here with permission.)

It's good that defining algorithms is out of scope, but my experience is that the use of an algorithm requires a few words of explanation, much as https://w3c-ccg.github.io/security-vocab/#Ed25519VerificationKey2018 and https://tools.ietf.org/html/rfc8080#section-3 show "here's how we format the key".  Typically I want those definitions to be in their own documents, so it's (relatively) easy to add new ones.  (I also want them to be more well-defined than I see in https://w3c-ccg.github.io/security-vocab/)  Which brings us to:

Registries.  I haven't been following the [recent Process changes around registries](https://github.com/w3c/w3process/pull/335), but it looks like Linked Data Security Vocabulary (LDSV) is, in fact, a registry.  Only it's a registry that contains the above explanation (albeit only by example).  I think it might be better to create a registry - including [specifying update criteria](https://pr-preview.s3.amazonaws.com/w3c/w3process/335/3ae9f7c...ebd4c99.html#reg-def) - and then create docs that populate initial values.  In the IETF, it's possible to do both at once, creating the registry in the same doc that defines initial values (see the later paragraphs of https://tools.ietf.org/html/rfc5155#section-11: "This document creates a new IANA registry...")
 I don't know that w3c's registry stuff will look like. Perhaps we should say, as a work/scope item: the WG will create a registry and populate initial values, without being clear what that document is going to be? @plehegar ?

### LINKED ISSUE #47: Linked Data Signature or Linked Data Security?
(Originally raised by @samuelweiler in https://github.com/w3c/strategy/issues/262#issuecomment-822696701; moved here with permission.)

It seems like the Linked Data Security (LDS) spec should instead be titled Linked Data Signatures (or signing).  Please change the name or explain the choice.

### PR DESCRIPTION
**Pull request by iherman**
This PR implements the proposal in https://github.com/w3c/lds-wg-charter/issues/47#issuecomment-825749246 for a new division of the deliverables. I used the terms “Linked Data Hash” and “Linked Data Integrity” to name the two new documents, following the proposal of @OR13 and @gkellogg, respectively.

I hope the new description of the two new deliverables (replacing the old one), reflects the thoughts of @dlongley and @msporny.

Beyond those changes (and some other editorial changes) I have also adopted the proposal in https://github.com/w3c/lds-wg-charter/issues/46#issuecomment-823945189 on registries. The relationship of the Linked Data Vocabulary (normative) and Linked Data Registry (non-normative) deliverables is indeed not 100% clear, also in view of the process changes, so better treat these two issues together. In so doing I have added a reference to the possible new process; as the charter proposal will undergo W3M review, eventually, that review will tell us whether keeping this in the charter text is appropriate or not (or propose an alternative formulation).

CC: @gkellogg @dlongley

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/lds-wg-charter/pull/56.html" title="Last updated on May 1, 2021, 5:57 PM UTC (7d85716)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/lds-wg-charter/56/96dd65d...7d85716.html" title="Last updated on May 1, 2021, 5:57 PM UTC (7d85716)">Diff</a>

### CODE REVIEW

File: index.html
Comment (msporny) [inline index.html:281]: To start -- I won't object to this; I could live with it.

I do feel like it's unnecessary and the description here could be folded into the Linked Data Integrity specification. While it's true that the steps are 1) canonize, 2) hash, 3) sign/protect -- step 2 is a 1 page specification, if we choose to split it out. I don't think it's necessary to split it out if we choose "Linked Data Integrity".

Comment (iherman) [reply index.html]: You are probably right that this 'LDH' spec might be very short. However, the preamble of this section does say:

> Note that the titles of the documents are tentative and not final. The Working Group may also decide to either split some of those documents or, conversely, merge some of them. 

So, at the end of the day, we may decide to merge (or further subdivide) some of these planned document. 

For the time being, I see this list of deliverables in the charter at least as much as a way of clearly separating the technical "chapters" of a large, conceptual deliverable, as a list of specific documents that the WG will, eventually, publish. The discussions we were having so far has convinced me that this separation is really necessary to avoid further discussions when the charter ends up being voted upon.

**Review by gkellogg — APPROVED** [review]
The substance is great. There are some minor grammatical changes I would suggest, but I won’t be at a computer before later on Tuesday.

File: index.html
Comment (dlongley) [reply index.html]: The WG will have flexibility here with naming and shuffling things around, so I'm ok with it -- we're just trying to capture the concepts and I think this does.

File: index.html
Comment (dlongley) [inline index.html:298]:
```suggestion
                            This specification defines a generic framework expressing proofs of integrity of Linked Datasets via, for example, digital signatures, proofs of work, or proofs of existence. Beyond the generic (normative) framework, this deliverable also provides a normative definition for Linked Data Signatures, as well as means to express these proofs in an RDF Dataset in a way that enables a 3rd party to verify the integrity of the data.
```

File: index.html
Comment (dlongley) [inline index.html:311]: It may turn out that the context defined may not actually be preferred -- but just one example that could be used. It may also be that a number of contexts are defined or a process is recommended for defining them with narrowly scoped terms. Trying to give flexibility here.

```suggestion
                            This specification defines a formal RDF Vocabulary for the terms defined in the Linked Data Integrity deliverable. The specification may also define one or more <a href="https://www.w3.org/TR/json-ld11/#the-context">JSON-LD Context</a> documents to be used by a JSON-LD serialization.
```

**Review by dlongley — APPROVED** [review]
Approving with some minor nits.

**Review by pchampin — APPROVED** [review]
I like these clarifications, even if, as stated by @iherman and @dlongley, the WG may decide later to merge of of the deliverables.

**Comment by iherman** [comment]
@samuelweiler, we would certainly need your command and, hopefully, approval; these are changes initiated by your (justified!) comments...

File: index.html
Comment (gkellogg) [inline index.html:298]:
```suggestion
                            This specification defines a generic framework for expressing proofs of integrity of Linked Datasets via, for example, digital signatures, proofs of work, or proofs of existence. Beyond the generic (normative) framework, this deliverable also provides a normative definition for Linked Data Signatures, as well as a means to express these proofs in an RDF Dataset in a way that enables a 3rd party to verify the integrity of the data.
```

**Review by philarcher — APPROVED** [review]
I agree that splitting out the hash from the integrity deliverable seems a little over the top but it does add clarity to the charter. I'm reasonably sure we'll make it a section within the integrity spec but that's a decision for later.

Overall, this looks good. The level and variety of support the charter is receiving is very positive.

**Comment by iherman** [comment]
> I'm reasonably sure we'll make it a section within the integrity spec but that's a decision for later.

Actually, I was thinking about merging this with the canonicalization document :-)

Some bikeshedding for the future!

**Comment by iherman** [comment]
@samuelweiler I'd prefer to merge this PR but keep the original issues open, waiting for you further comments. It makes other, parallel charter discussion less complicated...

**Comment by samuelweiler** [comment]
Fine with me.

**Pull request merged**
